lib/handlebars_helpers.js

```diff
@@ -0,0 +1,7 @@
+var Handlebars = require('handlebars');
+
+Handlebars.registerHelper('address_string', function(object) {
+  var addressArr = [object.city, object.region, object.countryCode];
+  addressArr = addressArr.filter(function(elem) {return elem != undefined});
+  return addressArr.join(', ');
```

Comment:
Sweet. much cleaner.
Be honest, did this quickly at lunch time and didnt even realize we already had underscore.
I'll commit the changes and make a new pull request.